Pages build vignette

mebowers5 opened the pull request:
Changed vignette to false eval on cluster. Loaded res.Rda from system file to lapply chunks in model the effects vignette section. Removed precompile vignettes from pkg and .Rbuildignore.

jdpye commented:
This is so close now! Looks like that underscore in the log filenames is your last showstopping problem?

mebowers5 commented:
Added false eval to failing vignette chunks that ran clean from user's perspective, resolved GitHub dependencies and updated README and documentation, built vignette with intent to push to GitHub pages
